code/modules/mob/mob_movement.dm

```diff
@@ -147,11 +147,10 @@
 
 	//We are now going to move
 	moving = 1
-	move_delay = mob.movement_delay() + world.time
+	move_delay = world.time

```

Comment:
Alright look this is great and all but what if
`move_delay = world.time - mob.movement_delay()`

Real talk here